calculator.js

```diff
@@ -265,18 +278,246 @@ input_element.addEventListener("click", event =>{
 //CALCULATOR
 function calculator(button){
     if(button.type == "operator"){
```

Comment:
You should have an OR here instead of two ifs where the action below them does the same

Reply:
I did it buddy